vtescrypt.csv

```diff
@@ -695,15 +695,15 @@
 "200596","Hezekiah Rutledge","","Vampire","Brujah","","4","5","ani cel pot PRE","Camarilla.","Gehenna:U","","","Christopher Shy"
 "200597","High Priest Angra Mainyu","","Vampire","Baali","","2","8","dem ser DAI OBF PRE","Independent: While Angra is ready, your prey gets -1 hand size. During a non-bleed action, he can burn the Edge to get +1 stealth. Infernal.","BL:U2, POD:DTC","","","Christopher Shy"
 "200598","Hillanvale","","Vampire","Daughter of Cacophony","","6","5","obf FOR MEL","Independent: During a bleed action, Hillanvale can burn 1 blood and discard a card requiring Melpominee [mel] to get +1 bleed. She gets -1 intercept against non-Independent, non-Anarch vampires.","HttB:U/B2, POD:DTC","","","Ken Meyer, Jr."
-"200599","Hiram ""Hide"" DeVries","","Vampire","Ventrue","","5","3","cel pre","Camarilla: While Hiram is ready, actions to burn a location you control cost 1 additional blood.","KoT:B2","","","Mattias Tapia"
+"200599","Hiram ""Hide"" DeVries","","Vampire","Ventrue","","5","3","cel pre","Camarilla: While Hiram is ready, actions to burn a location you control cost 1 additional blood.","KoT:B2","","","Matias Tapia"
 "200600","Homa","","Vampire","Osebo","","4","6","AUS CEL POT","Laibon.","LoB:U","","","Ken Meyer, Jr."
 "200601","Honest Abe","","Vampire","Brujah","","5","2","pre","Camarilla.","KoT:V/A1","","","Marian Churchland"
 "200602","Honorine Ateba","","Vampire","Ishtarri","","4","6","cel nec FOR PRE","Laibon.","LoB:PI2","","","Rebecca Guay"
 "201549","Horace Radcliffe","","Vampire","Nosferatu","","6","5","obf ANI POT","Camarilla.","V5:PN1, NB:PN1","","","Samuel Araya"
 "200603","Horatio","","Vampire","Tzimisce","","2","2","vic","Sabbat.","Sabbat:V, SW:U/PT, POD:DTC","","","Jeff Miracola"
 "200604","Horatio Ballard","","Vampire","Ventrue","","3","7","aus DOM FOR PRE","Camarilla: If Horatio doesn't have a title, he can call a referendum to become the Prince of Chicago as a +1 stealth political action.","CE:V, BSC:X","","","Mike Huddleston"
 "200605","Horde, The","","Vampire","Baali","","6","3","dai obf pre","Independent: During a bleed action, The Horde can lock to give another acting infernal minion +1 bleed. Infernal. Non-unique.","HttB:U2/A3, POD:DTC","","","James Stowe"
-"200606","Horrock","","Vampire","Gangrel","","3","6","vic ANI PRO","Independent: Protean [pro] cards cost Horrock 1 less blood.","Anarchs:U2","","","Matthew Mitchell"
+"200606","Horrock","","Vampire","Gangrel","","3","6","vic ANI PRO","Independent: Protean [pro] cards cost Horrock 1 less blood.","Anarchs:U2","","","Matt Mitchell"
```

Comment:
I would suppose, the other way around nowadays (seeing his insta: https://www.instagram.com/matthewmitchellart/ )